openfe/protocols/openmm_afe/base.py

```diff
@@ -272,21 +278,51 @@ def _get_system_generator(
         if ffcache is not None:
             ffcache = self.shared_basepath / ffcache
 
-        system_generator = system_creation.get_system_generator(
-            forcefield_settings=settings['forcefield_settings'],
-            integrator_settings=settings['integrator_settings'],
-            thermo_settings=settings['thermo_settings'],
-            cache=ffcache,
-            has_solvent=solvent_comp is not None,
-        )
+        # Block out oechem backend to avoid any issues with
+        # smiles roundtripping between rdkit and oechem
+        with without_oechem_backend():
```

Comment:
Here and everywhere else, to avoid completely removing access to oechem as a backend (e.g. for partial charge assignment) we instead block it out from all places where system_generator is used.

I _think_ `get_system_generator` is actually probably safe without, but it also doesn't cost us anything to be a bit extra cautious.